lib/src/integrationTest/kotlin/com/lemonappdev/konsist/core/declaration/kopsideclaration/forhasvalidkdoc/KoPsiDeclarationForHasValidKDocVerifyAuthorTagTest.kt

```diff
@@ -0,0 +1,42 @@
+package com.lemonappdev.konsist.core.declaration.kopsideclaration.forhasvalidkdoc
```

Comment:
Is this hierarchy ok in PsiDeclarationTest?

<img width="534" alt="image" src="https://github.com/LemonAppDev/konsist/assets/111683562/3e0e6067-7650-401e-bfdc-f91ab18b866a">